### PR TITLE
Classes Review

### PR DESCRIPTION
**Pull request by Felipeg005**
In this branch, we add the next features:

1. use JavaScript classes instead of objects and arrays.
2. Use inside classes arrays.
3. Create class methods to add and remove books.
4. Maintain all of the original functionality of the application (i.e. no bugs were introduced while refactoring).
5. Don't use any library or framework, use only plain JavaScript.

Thank you for your review.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,78 +1,91 @@
-const books = [];
-const titlePlace = document.getElementById('title');
-const authorPlace = document.getElementById('author');
-const form = document.getElementById('form');
+/* eslint-disable */const form = document.getElementById('form');
```
Comment (carlos-ssh) [inline script.js]: - Kindly enable your linters.
**Check the way on how to use ESLint comments** [here](https://eslint.org/docs/2.13.1/user-guide/configuring).

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hello team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (Felipeg005) [reply script.js]: Ok, thank you, now the linters are enabled and without errors 😄

**Review by hemantso — APPROVED** [review]
Hi Team, 

Good work on making the requested changes. 🚀 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**